web/Services/PropertyService.cs

```diff
@@ -99,4 +101,18 @@ public void DeleteProperty(PropertyDbModel property)
         return _dbContext.Properties.SingleOrDefault(p => p.Id == propertyId);
     }
 
+    public bool IsUserAdminOrCorrectLandlord(BricksAndHeartsUser currentUser, int propertyId)
+    {
+        if (currentUser.IsAdmin)
+        {
+            return true;
+        }
+        var propertyLandlordId = GetPropertyByPropertyId(propertyId)!.LandlordId;
+        var userLandlordId = currentUser.LandlordId;
+        if (propertyLandlordId == userLandlordId)
+        {
+            return true;
+        }
+        return false;

```

Comment:
Can shorten to
```suggestion
        return propertyLandlordId == currentUser.LandlordId;;
```
think just as easy to understand